integration-tests/rpc-nodes.spec.ts

```diff
@@ -399,17 +388,17 @@ CONFIGS().forEach(
           done();
         });
 
-        jakartanetIt('getTxRollupInbox', async (done) => {
+        it('getTxRollupInbox', async (done) => {
           const inbox = await rpcClient.getTxRollupInbox('txr1YTdi9BktRmybwhgkhRK7WPrutEWVGJT7w', '0');
           expect(inbox).toBeDefined();
           done();
         });
 
-        jakartanetIt('getTxRollupState', async (done) => {
-          const state = await rpcClient.getTxRollupState('txr1YTdi9BktRmybwhgkhRK7WPrutEWVGJT7w');
-          expect(state).toBeDefined();
-          done();
-        });
+        // it('getTxRollupState', async (done) => {
```

Comment:
Why is this test commented?

Reply:
waiting for the endpoint to be re-activated. But I understand the test needs a new rollup address, so  I will fix it.

Reply:
repaired the test by splitting the j and K tests to each use its an appropriate rollup address